.idea/workspace.xml

```diff
@@ -0,0 +1,491 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
unnecessary file. Please list all such folders in .gitignore file

Reply:
I added .idea to the .gitignore file